components:
- !!python/object:uuid.UUID
  int: 170479705391647032501098302123768246956
created_by: &id001 !!python/object:uuid.UUID
  int: 217356799640352244068158184369341070513
created_time: 2010-02-26 10:03:45.042625
description: 'After I used pitz-destroy to get rid of a task, I ran

  pitz-recent-activity, and it crashed.


  I suspect that some activities pointed to the destroyed task.'
estimate: !!python/object:uuid.UUID
  int: 207205260775652781422748338617572801073
milestone: !!python/object:uuid.UUID
  int: 43202488294965147244317640483158997306
modified_time: 2010-03-13 08:21:26.418129
owner: *id001
pscore: 0
status: !!python/object:uuid.UUID
  int: 181191650815078915740523248343325601886
title: Make pitz-destroy clean up related activity entities
type: task
uuid: !!python/object:uuid.UUID
  int: 215358700129406542433302315512004491529
yaml_file_saved: 2010-03-13 08:21:26.432444
